game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyLogin.java

```diff
@@ -98,6 +98,11 @@ private IMessenger login(final String userName, final String password, final boo
         });
   }
 
+  private static String playerMacIdString() {
+    final String mac = MacFinder.getHashedMacAddress(new byte[6]);
```

Comment:
I think you want the no-arg overload here.  This overload simply hashes the passed array, which means it will always hash the MAC `00-00-00-00-00-00`.

Reply:
Thanks, fixed in: ea97854